### DISCUSSION TITLE
Getting Max retries exceeded with url error while using the colab code

### DISCUSSION THREAD

**Discussion by SaurabhRajpoot1** [ShishirPatil/gorilla#441] [Q&A] [answered]
I am using Google Colab code, i am getting this error ->  Error communicating with OpenAI: HTTPConnectionPool(host='luigi.millennium.berkeley.edu', port=8000): Max retries exceeded with url: /v1/chat/completions (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ed4cc520c40>: Failed to establish a new connection: [Errno 111] Connection refused')) gorilla-openfunctions-v0 What's the weather like in the two cities of Boston and San Francisco?
any help

**Comment by SaurabhRajpoot1** [comment]
please answer if any one used

**Comment by ShishirPatil** [comment, marked as answer]
Hey folks, can you try with the following url: `url = "https://luigi.millennium.berkeley.edu/v1/chat/completions` Thank you!

Fix: https://github.com/ShishirPatil/gorilla/pull/560

**Reply by ShishirPatil** [reply]
Also make sure you have `pip install openai==0.28`
